### DISCUSSION TITLE
Is there a way to test from different devices locally?

### DISCUSSION THREAD

**Discussion by emclaren** [mozilla/hubs#2724] [General Questions & Discussion] [answered]
I'm trying to access my local server from another computer and I'm getting a cors issue
I'm running the front end locally and pointing to our own instance of reticulum.
if I use localhost:8080 everything works fine.
but if I want to test it from another computer or using my oculus go I get an CORS error from the browser.
it seems the back end is not allowing the request for another address that is not the same domain and localhost which seems to be expected, but I would like to know if there's a way for me to test this locally without having to constantly deploy the code.

Note: copied from Discord

**Comment by robertlong** [comment, marked as answer]
You can use the Chrome inspect tab (chrome://inspect) and test using port forwarding. You'll need to forward port 8080 and then go to https://localhost:8080 on your device.

https://developers.google.com/web/tools/chrome-devtools/remote-debugging/local-server

**Reply by netpro2k** [reply]
More detailed instructions for testing on an Oculus Quest specifically can also be found here: https://github.com/mozilla/hubs/blob/master/CONTRIBUTING.md#6-testing-on-an-hmd

**Reply by johnny-carmona** [reply]
I believe this might not work. Since the device won't be running the app. The scenario I'm referring to is running the app on one computer and then accessing the app through the local network using the oculus go or something else pointing to the computer that is running the app.

**Reply by joshmarinacci** [reply]
Generally you will want to run your testing version of hubs either on a webserver with a real cert (not self-signed) or through a proxy that gives you that.  Browsers, especially mobile device browsers, will have lots of issues with non-self-signed certs.  I personally use the service [ngrok.com](https://ngrok.com/)  to create a public https end-point for the hubs client running on my desktop. Then I can connect to it from anything.
